### PR TITLE
Supports exports of set of ltext

### LINKED ISSUE #124: Support export mapping for set of ltext attributes
Hey there,

currently I have a customer who is using a set of ltext attributes. It seems, that import mapping of these kind of attributes is already covered (https://github.com/sphereio/sphere-node-product-csv-sync/blob/master/src/spec/mapping.spec.coffee#L314) but the export mapping is missing here. For localized fields, this seems to be the only case currently missing (https://github.com/sphereio/sphere-node-product-csv-sync/blob/master/src/spec/exportmapping.spec.coffee#L94-L96).

Please also support the attribute mapping of sets of ltext for exports.

Thanks.

### PR DESCRIPTION
**Pull request by hisabimbola**
Fixes #124

### CODE REVIEW

File: src/spec/exportmapping.spec.coffee [hunk 1]
```diff
@@ -344,6 +344,86 @@ describe 'ExportMapping', ->
...
+              {"en": "foo2", "de": "barB"},
+              {"de": "barC", "nl": "invisible"}
+            ]
+          }
+        ]
+      row = @exportMapping._mapVariant(variant, @productType)
+      expected = [ undefined,'foo1;foo2', 'barA;barB;barC' ]
+      expect(row).toEqual expected
+
+    it 'should ignore trailing invalid set of ltext', ->
```
Comment (PhilippSpo) [inline src/spec/exportmapping.spec.coffee]: Don't get me wrong here, I really like that you added these tests. They should just be independent of whether there is a set attribute or any other attribute.

Comment (hisabimbola) [inline src/spec/exportmapping.spec.coffee]: Yeah, I figured that it's a general behaviour and not specific to _ltext_. Updating the description of the test will make it more appropriate I guess.

**Comment by hajoeichler** [comment]
LGTM, but I would recommend (maybe also after merging) to enhance the import-export cucumber acceptance test to include set of ltext there too

File: src/spec/exportmapping.spec.coffee [hunk 2]
```diff
@@ -344,6 +344,86 @@ describe 'ExportMapping', ->
...
+              {"en": "foo2", "de": "barB"},
+              {"de": "barC", "nl": "invisible"}
+            ]
+          }
+        ]
+      row = @exportMapping._mapVariant(variant, @productType)
+      expected = [ 'barA;barB;barC','foo1;foo2' ]
+      expect(row).toEqual expected
+
+    it 'should return undefined for invalid set of attributes', ->
```
Comment (PhilippSpo) [inline src/spec/exportmapping.spec.coffee]: Well, these tests still belong the `describe '#mapLtextSet'`, but OK.

**Pull request merged**